### ISSUE TITLE
KaTeX JS not included by default

### ISSUE THREAD

**Issue by darsnack** [MichaelHatherly/Publish.jl#6]
None of the math blocks are rendered using KaTeX in my deployed page (see below). Is the JS loaded by default, or is there something required on my part? I see the resulting HTML is wrapped in `<div class="math">`.
 
<img width="773" alt="Screen Shot 2020-12-24 at 6 46 51 PM" src="https://user-images.githubusercontent.com/4811350/103112131-dfa19480-4618-11eb-8a14-2c703803a99a.png">

**Comment by MichaelHatherly** [comment]
There's no included math rendering lib in the HTML output at the moment. I'm happy to have one included by default, KaTeX is probably the best default to use.

**Comment by darsnack** [comment]
It seems like KaTeX expects the class `"katex"`, but the markdown generates the classes `"math"` and `"display-math"`.

**Comment by darsnack** [comment]
Never mind, it seems as though that is irrelevant for rendering.

**Comment by MichaelHatherly** [comment]
> It seems like KaTeX expects the class "katex", but the markdown generates the classes "math" and "display-math".

Yup, the markdown->html rendering is meant to be reasonably agnostic with respect to the libraries you end up using in the final document, so you have to point katex to the specific nodes with classes math and display-math.

Currently there's some implementation code in https://github.com/MichaelHatherly/PublishThemes/blob/master/themes/default/html_doc.mustache which is used for the (currently) undocumented self-contained `html_doc` output format. I just need to port over that katex stuff to the main `html` theme as well.

**Comment by darsnack** [comment]
Yeah using the event listener is the approach I ended up using too.

**Comment by kapple19** [comment]
Hi all, can someone ELI5 how to activate the KaTeX rendering for HTML output using `Publish@0.10.0`? Passing `html_doc` to `serve` throws an error.
